backend/store/store.go

```diff
@@ -99,14 +99,18 @@ type SelectionPredicate struct {
 	// returned empty from the store, it indicates that there's no additional
 	// resources available
 	Continue string
+	// Offset indicates the index of the first resource to retrieve
+	Offset int64
```

Comment:
Did we decide if we wanted to do this or not?

Reply:
never mind me

Reply:
My take is that we shouldn't need this field so you can remove it.

Reply:
Unless I'm mistaken, my only concern is that the resolver would need to fabricate a continue token